File: CPP/jakmasznaimie.cpp
/*
 * jakmasznaimie.cpp
 * 
 * 
 */


#include <iostream>
using namespace std;   

int parzyste(int a){
    int i;
    for (i=0; i<=a; i+=2)
        cout << i << " " ;
    return i/2;
        
}
    
        
    
  

int main(int argc, char **argv)
{
    string imie;
    int wiek;
    int p;
    int a;
    
    cout << "Heloł" << endl;
    cout << "Podaj swoje imię" << endl;
    getline(cin,imie);
    cout << "Heloł " << imie << endl;
    cout << "Ile masz lat?" << endl;
    cin >> wiek;
    cout << "Urodziles sie w " << 2017-wiek <<  " roku.";
    p=26;
    if(wiek<p){
        cout << "Jestem starszy!" << endl;
        }else if(wiek==p){
            cout << "Mamy tyle samo lat!" << endl;
            }else
        cout << "Jestem mlodszy!" << endl;
    
    cout << "Podaj liczbe" << endl;
    cin >> a;
    cout << "Parzystych liczb jest : " << parzyste(a) << endl;
    
    
    //for(int i=0;i<=100;i+=2){
     //   cout << i << endl;
        
        
        
        

    return 0;
}

File: PAJTON/jakmasznaimie.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  jakmasznaimie.py 

def parzyste(n):
    ile=list(range(0,n+1,2))
    print(ile)
    return len(ile)

def main(args):
    return 0

if __name__ == '__main__':
    
    print("helo")
    
    imie=input("jak masz na imie?")
    print("hello ",imie,)
    wiek=int(input("ile masz lat?"))
    
    print("Urodziles sie w ", 2017-wiek)
    
    c=1991
    rok=2017-wiek
    
    if c==rok:
            print("mam tyle lat co Ty!")
    elif c<rok:
            print("Jestem od Ciebie starszy!")
    elif c>rok:
            print("Jestem młodszy od Ciebie!")
    n=int(input("podaj liczbe : "))
    print("parzystych : ", parzyste(n))
    
    
    import sys
    sys.exit(main(sys.argv))
